shared/common-adapters/labeled-input.tsx

```diff
@@ -149,7 +149,7 @@ const styles = Styles.styleSheetCreate(
         paddingLeft: Styles.globalMargins.xsmall,
         paddingRight: Styles.globalMargins.xsmall,
         width: '100%',
-        zIndex: 10,
+        zIndex: 0,
```

Comment:
Not sure what's going on here, but this doesn't seem to be necessary, and it was messing up overlays.